tst/regression/test_suites/particle_leapfrog_outflow/particle_leapfrog_outflow.py

```diff
@@ -32,12 +32,18 @@ def Prepare(self, parameters, step):
         return parameters
 
     def Analyse(self, parameters):
+        sys.path.insert(
+            1,
+            parameters.parthenon_path
+            + "/scripts/python/packages/parthenon_tools/parthenon_tools",
```

Comment:
Should we insert this path only if an installed parthenon_tools isn't found? Not sure the right way to do that.

Reply:
For regression tests, I think it's correct to always use the version of `parthenon_tools` packaged with the source code, since the tests should be testing that?

Reply:
Yeah you're right -- I see now too that we do this for all the test suites